app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationList.jsx

```diff
@@ -128,6 +130,8 @@ export default function RegistrationAdministrationList({ competitionInfo }) {
 
   const actionsRef = useRef();
 
+  const [autoAcceptEnabled, setAutoAcceptEnabled] = useState(competitionInfo.auto_accept_registrations)
```

Comment:
Can we use `competitionInfo` as the source of truth?

Reply:
Currently it is being hydrated on render (not queried via API), so there's no way to refetch it currently. Assuming we could refetch it, how would you suggest we then update whether or not the "Disable auto accept" button is rendered? (my understanding is that React doesn't detect changes in the state of variables)

Reply:
if we would fetch it, we would rely on the react Query to handle the state being up to date.

Reply:
Yeah it would be pretty simple if we switch to fetching it with react query, I think.

Having a situation where there are 2 sources of truth (`competitionInfo.auto_accept_registrations` and `autoAcceptEnabled`), and which could disagree, is not good - leaves room for future bugs, etc.